website/src/components/SequenceDetailsPage/SequencesContainer.tsx

```diff
@@ -49,75 +49,118 @@ export const InnerSequencesContainer: FC<SequenceContainerProps> = ({
+            organism={organism}
+            accessionVersion={accessionVersion}
+            clientConfig={clientConfig}
+            nucleotideSegmentNames={nucleotideSegmentNames}
+            sequenceType={sequenceType}
+            setType={setSequenceType}
+            genes={genes}
+        />
     );
 };
 
 export const SequencesContainer = withQueryProvider(InnerSequencesContainer);
 
-type NucleotideSequenceTabsProps = {
+type SequenceTabsProps = {
+    organism: string;
+    accessionVersion: string;
+    clientConfig: ClientConfig;
     nucleotideSegmentNames: NucleotideSegmentNames;
     sequenceType: SequenceType;
     setType: Dispatch<SetStateAction<SequenceType>>;
+    genes: string[];
 };
 
-const SequenceTabs: FC<NucleotideSequenceTabsProps & { genes: string[] }> = ({
+const SequenceTabs: FC<SequenceTabsProps> = ({
+    organism,
+    accessionVersion,
+    clientConfig,
     nucleotideSegmentNames,
     genes,
     sequenceType,
     setType,
-}) => (
-    <BoxWithTabsTabBar>
-        <UnalignedNucleotideSequenceTabs
-            nucleotideSegmentNames={nucleotideSegmentNames}
-            sequenceType={sequenceType}
-            setType={setType}
-        />
-        <AlignmentSequenceTabs
-            nucleotideSegmentNames={nucleotideSegmentNames}
-            sequenceType={sequenceType}
-            setType={setType}
-        />
-        {genes.map((gene) => (
-            <BoxWithTabsTab
-                isActive={isGeneSequence(gene, sequenceType)}
-                onClick={() => setType(geneSequence(gene))}
-                label={gene}
-                key={gene}
-            />
-        ))}
-    </BoxWithTabsTabBar>
-);
+}) => {
+    const [activeTab, setActiveTab] = useState<'unaligned' | 'aligned' | 'gene'>('unaligned');
+
+    useEffect(() => {
+        if (isUnalignedSequence(sequenceType)) {
+            setActiveTab('unaligned');
+        } else if (isAlignedSequence(sequenceType)) {
+            setActiveTab('aligned');
+        } else if (isGeneSequence(sequenceType.name, sequenceType)) {
+            setActiveTab('gene');
+        }
+    }, [sequenceType]);
+
+    return (
+        <>
+            <BoxWithTabsTabBar>
+                <UnalignedNucleotideSequenceTabs
+                    nucleotideSegmentNames={nucleotideSegmentNames}
+                    sequenceType={sequenceType}
+                    setType={setType}
+                    isActive={activeTab === 'unaligned'}
+                    setActiveTab={setActiveTab}
+                />
+                <AlignmentSequenceTabs
+                    nucleotideSegmentNames={nucleotideSegmentNames}
+                    sequenceType={sequenceType}
+                    setType={setType}
+                    isActive={activeTab === 'aligned'}
+                    setActiveTab={setActiveTab}
+                />
+                <BoxWithTabsTab
+                    isActive={activeTab === 'gene'}
+                    label='Amino acid sequences'
```

Comment:
Shoud we rather name it "Aligned amino acid sequences" which would be more accurate? It might be worth emphasizing this so that users know that insertions are not included.